### ISSUE TITLE
Add the new lr_scheduler which called poly

### ISSUE THREAD

**Issue by acgtyrant** [pytorch/pytorch#2854]
The learning policy is from [DeepLab v2](https://arxiv.org/abs/1606.00915):

> Learning rate policy: We have explored different learning rate policies when training DeepLab-LargeFOV. Similar to [86], we also found that employing a “poly” learning rate policy (the learning rate is multiplied by (1􀀀 iter max iter )power) is more effective than “step” learning rate (reduce the learning rate at a fixed step size).

If you agree, I open a pull request later.
